test/tests/mesh/mesh_only/tests

```diff
@@ -8,6 +8,15 @@
     method = 'opt oprof'
   [../]
 
+  [./mesh_info]
+    type = 'RunApp'
+    input = 'mesh_only.i'
+    cli_args = '--mesh-only info_test.e'
+    recover = false
+    method = 'opt oprof'
+    expect_out = 'Mesh Information'
```

Comment:
This is missing `design`, `issue` and `requirement` parameters.  I checked and there is no `MeshOnlyAction.md`, so I assuming we need one?